code/datums/mutantraces.dm

```diff
@@ -232,6 +232,11 @@ ABSTRACT_TYPE(/datum/mutantrace)
 
 	var/self_click_fluff //used when clicking self on help intent
 
+	/// Abilityholder associated with this mutantrace, will be automatically given to mobs on spawn
+	var/datum/abilityHolder/mutant_abilityholder = null
```

Comment:
```suggestion
	/// Abilityholder associated with this mutantrace, will be automatically given to mobs on spawn
	var/mutant_abilityholder = null
```
Comment should make it clear this is a type, otherwise it looks like an instance var.

Reply:
why remove the type

Reply:
Because it's a lie!! When you see a var typed as `/datum/abilityHolder` you expect it to contain an instance of `/datum/abilityHolder` not a type. It's not like typing it offers any validation that the type you put in there will inherit from the correct parent.